### DISCUSSION TITLE
[Q&A] `peronitrous acid` SMILES representation with 3 different conformer?

### DISCUSSION THREAD

**Discussion by IchiruTake** [rdkit/rdkit#4949] [Q&A]
How to represent the `peronitrous acid` by three different conformers by SMILES?

I found the basic structure to be `OON=O` but is stuck to finding the SMILES of these three different conformers? trans-perp, cis-cis, and cis-perp. I try out chirality `[@]` and `[@@]` but getting stuck with double bond and these chirality.

DOI: 10.1063/1.467338

**Comment by dehaenw** [comment]
I'm not sure if SMILES representation can distinguish these, because they are conformations and not isomers of the same structure and the difference is rotations around freely rotatable single bonds. So these different conformations would have to be dealt with after embedding or at some point when you have access to the bond angles, imo.
